SPARQLWrapper/SmartWrapper.py

```diff
@@ -153,11 +151,12 @@ def getValues(self, key):
         :rtype: list
         """
         try:
-            return [b[key] for b in self[key]]
-        except:
-            return []
+            vals = self[key]
+            return [val[key] for val in vals]
+        except Exception:
+            return None
```

Comment:
Looks like this can go into a separate PR.

Reply:
934e334